samples/client/petstore/javascript-flowtyped/README.md

```diff
@@ -0,0 +1,40 @@
+## swagger_petstore@1.0.0
+
+This generator creates Flow typed JavaScript client that utilizes [Fetch API](https://fetch.spec.whatwg.org/). The generated Node module can be used in the following environments:
+
+Environment
+* Node.js
```

Comment:
Could anyone help provide a guideline for how to get Node.js to work with this flavour of JavaScript client? At the time of writing, Node.js doesn't support ES6 module system yet. Is there a codegen configuration we could use for the node.js environment? Thanks.

Reply:
just run `npm install && NODE_ENV=development node run build` to transpile into valid es5 code.